Fix "by using" in concepts A--M (#3206)
theforeman/foreman-documentation: guides/common/modules/con_composite-content-views-overview.adoc

.Repository restrictions
Docker repositories cannot be included more than once in a composite content view.

> For example, if you attempt to include two content views by using the same docker repository in a composite content view, {ProjectServer} reports an error.

Reviewer: alternative:
```suggestion
For example, if you attempt to include two content views with the same docker repository in a composite content view, {ProjectServer} reports an error.
```

or "... that contain ..."

Author: ```suggestion
For example, if you attempt to include two content views that contain the same docker repository in a composite content view, {ProjectServer} reports an error.
```
